### DISCUSSION TITLE
Header and sidebar stacked layout

### DISCUSSION THREAD

**Discussion by kachidk** [themesberg/flowbite#194] [Ideas]
A header and sidebar stacked layout would be a beautiful addition to Flowbite blocks, for those building dashboard UIs.

**Comment by zoltanszogyenyi** [comment]
Hey @kachidk,

Right now we have a sidebar + header layout for the pro version's [application UI dashboard](https://flowbite.com/application-ui/preview/) but we might add an example to the free version in the future!
